projects/spectator/test/events/events.component.spec.ts

```diff
@@ -58,7 +58,7 @@ describe('EventsComponent', () => {
     try {
       spectator.keyboard.pressKey('...', 'input');
       fail('Expected that an error is thrown');
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
TypeScript 4.4 defaults to `e` being `unknown`.
Previously it was `any` by default. This change just makes that explicit.